plans/timeout_plan.fmf

```diff
@@ -0,0 +1,10 @@
+summary: Basic timeout test
+execute:
+    how: tmt
+    duration: 10s
```

Comment:
```suggestion
    duration: 60s
```
Shouldn't this be 60 seconds? Because the sleep takes 60 seconds.

Reply:
No, because we need test, that TEsting Farm really timeouts.